monitoring/cerberus/Dockerfile

```
@@ -1,6 +1,6 @@
-FROM image-registry.openshift-image-registry.svc:5000/openshift/cli
+# FROM image-registry.openshift-image-registry.svc:5000/openshift/cli
 # Use this when building off cluster
-#FROM registry.redhat.io/openshift4/ose-cli:v4.5
+FROM registry.redhat.io/openshift4/ose-cli:v4.5
```

```suggestion
FROM registry.redhat.io/openshift4/ose-cli:v4.6
```
LABs are 4.6 and Silver is upgrading next week